packages/capabilities/test/capabilities/consumer.test.js

```diff
@@ -0,0 +1,165 @@
+import assert from 'assert'
+import { access } from '@ucanto/validator'
+import { Verifier } from '@ucanto/principal/ed25519'
+import * as Consumer from '../../src/consumer.js'
+import { alice, bob, service, mallory } from '../helpers/fixtures.js'
+import { parseLink } from '@ucanto/core'
+import { createCborCid } from '../helpers/utils.js'
+
+describe('consumer capabilities', function () {
+  describe('consumer/add', function () {
```

Comment:
I think this could use a test ensuring that [this example](https://github.com/web3-storage/specs/blob/main/w3-provider.md?plain=1#L202) would be authorized, whose proof includes `consumer: did:*` but the invocation `att` has `consumer: did:key:zSpace`